internal/mikrotik/record.go

```diff
@@ -60,22 +60,28 @@ func NewRecordFromEndpoint(endpoint *endpoint.Endpoint) (*DNSRecord, error) {
 func NewEndpointFromRecord(record DNSRecord) (*endpoint.Endpoint, error) {
 	log.Debugf("converting Mikrotik DNS record: %v", record)
 
+	recType := record.Type
+	// "" means A record because mikrotik is weird like that... :P
+	if recType == "" {
+		recType = "A"
+	}
+
 	ep := endpoint.Endpoint{
 		DNSName:    record.Name,
-		RecordType: record.Type,
+		RecordType: recType,
 		// TODO: ttl
 		// TODO: ProviderSpecific
 	}
-	switch record.Type {
-	case "", "A", "AAAA": // "" means A record because mikrotik is weird like that... :P
+	switch recType {
```

Comment:
super nitpicky, i know, but I prefer to switch on `ep.RecordType` rather than rectype 😅